scripts/app_domain

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+
+app_domain = YAML.load_file("config/application.yml").fetch("APP_DOMAIN") do
```

Comment:
Using `.env` is a possibility, but that'd have `APP_DOMAIN` setup in two places: `application.yml` and `.env`. I believe that'd be bad developer experience.

Reply:
To be clear, `APP_DOMAIN` wasn't added by me, it already existed for whatever reason. So, adding it to `.env` as well would be duplication.

Reply:
While I do agree that it is clunky to have to set this domain twice, I want to avoid having one off scripts in `scripts/` when the information you need to set the correct domain in webpacker can be made available very simply by setting an env var. 

Sounds like we need to refactor some things in regards to config settings and secrets. For some context, myself and some coworkers are looking at how we are handling env vars and secrets etc, so we might be able to dedup this and more down the road.

Reply:
@ledestin would the "duplicate" env solve the problem or isn't it enough? Because if so I think we should lean that way. We're also in process of overhauling our secrets configurations for Forem so it might be okay to live with this duplication for a little while

Reply:
@jdoss @rhymes Thanks for the explanations.

I was blind to a better solution, namely just setting variables in `webpacker.yml`. Switched to that, please review.